tests/eas/generic.py

```diff
@@ -329,6 +329,15 @@ class ThreeSmallTasks(_EnergyModelTest):
     """
     Test EAS for 3 20% tasks over 2 seconds
     """
+
+    # The energy estimation for this test is probably not very accurate and this
+    # isn't a very realistic workload. It doesn't really matter if we pick an
+    # "ideal" task placement for this workload, we just want to avoid using big
+    # CPUs in a big.LITTLE system. So use a larger energy threshold that
+    # hopefully prevents too much use of big CPUs but otherwise is flexible in
+    # allocation of LITTLEs.
+    energy_est_threshold_pct = 20
```

Comment:
Moreover, perhaps this should be better an issue but, this test hardcode 3 small tasks... which will not fit on platforms like pixels for example.

Should not we fix these tests to make them more portable and less TC2 style.... we can still use the LISA's RTApp configuration to ask for a number of small/big tasks which exactly matches the little/big cores number... and update the tests names to be more aligned... for example, this should be s/ThreeSmallTasks/OnlySmallTasks/

What do you think? Should I open an Issue... or should we better (at least try to) fix workload compositions and tests names for this release?